IHR2-2798 refactor expand params (#233)
dotwebstack/dotwebstack-framework: service/openapi/README.md

```

# 1.1.4 Expand parameter
By default only GraphQL fields with the `ID` type and the fields that are marked as `required` in the OpenApi response 

> are returned. If a required field in OpenApi is of type `object` in GraphQL, the child fields of this type with the `ID` 

Reviewer: ```suggestion
are returned. If a required field in OpenApi is of type `object` in GraphQL, only the child fields of this type with the `ID` 
```

      enum: ['beers', 'beers.ingredients', 'beers.supplements']
```

In the example the expand parameter is used to include `beers` in the response by default. Since `beers` refers to an 

> object field in GraphQL, it means that the fields within `beers` with an `ID` type are returned as well, all other fields 

Reviewer: ```suggestion
object field in GraphQL, it means that only the fields within `beers` with an `ID` type are returned by default.
```
